docs/getting-started/overview.rst

```diff
@@ -1,6 +1,106 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several characteristics of .NET Core that make it special:
+
+**Cross-platform support** is the first important feature. For applications, it 
+is important to use those platforms that will provide the best environment for 
+their execution. Thus, having an application platform that can enable the app 
+to be ran on different operating systems with minimal or no changes provides a 
+significant boon. 
+
+**Open Source** because it is proven to be a great way to enable a larger set of 
+platforms, supported by community contribution.
+
+**Better packaging story** that allows the framework to be distributed 
```

Comment:
Yes, well, this was before it was made public, not my fault someone's been tardy with their PR feedback. ;) Jokes aside, will fix, good point, this is now a done deal.